python/core/auto_generated/qgis.sip.in

```diff
@@ -125,6 +125,41 @@ The development version
 A string with default project scales.
 
 .. versionadded:: 3.12
+%End
+
+    static int geosVersionInt();
```

Comment:
How would you feel about moving these to QgsGeometryEngine instead?

Reply:
I don't know. Indeed for GEOS it can be relevant to have this in a Geometry* class. But, what if we also add this kind of mechanism for other libraries (gdal, mdal, pdal...)?

Reply:
That's a fair point! I guess I just get nervous whenever the Qgis class is changed because it results in a super-slow recompilation :rofl:

Reply:
I know, I know... When I have to modify it I spend more time figuring out how to avoid touching it rather than pushing my changes on it. :rofl: